src/controllers/sign_up.js

```diff
@@ -1,3 +1,64 @@
-exports.get = (req,res) =>{
-    res.render('sign_up',{ style:"style", dom:"sign-up", title:"sign up",a:true});
+const hashPassword = require('./hashPassword');
+const { addUser } = require('../model/queries/users');
+const { addPerson } = require('../model/queries/person');
+const { addBusiness } = require('../model/queries/business');
+
+exports.get = (req, res) => {
+  res.render('sign_up', {
+    style: 'style', dom: 'sign_up', vald: 'validation', title: 'sign up', a: true,
+  });
+};
+
+exports.post = (request, response) => {
+  const data = request.body;
```

Comment:
these lines of code can be consolidated into one.
You don't need to assign ```request.body``` to a new variable.
You can declare the 3 values ```email, password and type``` in one destructuring statement and have it done directly to the ```req.body```